narwhals/dataframe.py

```diff
@@ -33,10 +33,10 @@
     from narwhals.typing import IntoDataFrame
     from narwhals.typing import IntoExpr
 
-FrameT = TypeVar("FrameT", bound="IntoDataFrame")
+IntoDataFrameT = TypeVar("IntoDataFrameT", bound="IntoDataFrame")
```

Comment:
we're using `FrameT` for both `DataFrame` and `LazyFrame`

maybe `DataFrame` should use `IntoDataFrameT` and `LazyFrame` should use `IntoFrameT`?